src/librustdoc/html/markdown.rs

```diff
@@ -303,7 +303,9 @@ impl<'a, I: Iterator<Item = Event<'a>>> Iterator for CodeBlocks<'_, 'a, I> {
                 attrs: vec![],
                 args_file: PathBuf::new(),
             };
-            let doctest = doctest::DocTestBuilder::new(&test, krate, edition, false, None, None);
+            let doctest = doctest::DocTestBuilder::new(
+                &test, krate, edition, false, None, None, None, DUMMY_SP,
```

Comment:
This needs a builder pattern, badly.

Reply:
Yep. Do you want it in this PR or in a follow-up?